# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# UTUMI Hirosi <utuhiro78@yahoo.co.jp>, 2019
# ブラシックデービッド, 2019
# Takefumi Nagata, 2019
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-04-15 07:27-0400\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Takefumi Nagata, 2019\n"
"Language-Team: Japanese (https://www.transifex.com/calamares/teams/20061/ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: src/modules/services-systemd/main.py:35
msgid "Configure systemd services"
msgstr "systemdサービスを設定"

#: src/modules/services-systemd/main.py:68
#: src/modules/services-openrc/main.py:102
msgid "Cannot modify service"
msgstr "サービスが変更できません"

#: src/modules/services-systemd/main.py:69
msgid ""
"<code>systemctl {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-systemd/main.py:72
#: src/modules/services-systemd/main.py:76
msgid "Cannot enable systemd service <code>{name!s}</code>."
msgstr "<code>{name!s}</code>というsystemdサービスが可能にすることができません"

#: src/modules/services-systemd/main.py:74
msgid "Cannot enable systemd target <code>{name!s}</code>."
msgstr "systemd でターゲット <code>{name!s}</code>が開始できません。"

#: src/modules/services-systemd/main.py:78
msgid "Cannot disable systemd target <code>{name!s}</code>."
msgstr "systemd でターゲット <code>{name!s}</code>が停止できません。"

#: src/modules/services-systemd/main.py:80
msgid "Cannot mask systemd unit <code>{name!s}</code>."
msgstr ""

#: src/modules/services-systemd/main.py:82
msgid ""
"Unknown systemd commands <code>{command!s}</code> and "
"<code>{suffix!s}</code> for unit {name!s}."
msgstr ""

#: src/modules/umount/main.py:40
msgid "Unmount file systems."
msgstr "ファイルシステムをアンマウントする。"

#: src/modules/unpackfs/main.py:40
msgid "Filling up filesystems."
msgstr ""

#: src/modules/unpackfs/main.py:158
msgid "rsync failed with error code {}."
msgstr "エラーコード {} によりrsyncを失敗。"

#: src/modules/unpackfs/main.py:219 src/modules/unpackfs/main.py:237
msgid "Failed to unpack image \"{}\""
msgstr "イメージ  \"{}\" の展開に失敗"

#: src/modules/unpackfs/main.py:220
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed"
msgstr "unsquashfs が見つかりませんでした。 squashfs-toolsがインストールされているか、確認してください。"

#: src/modules/unpackfs/main.py:313
msgid "No mount point for root partition"
msgstr "ルートパーティションのためのマウントポイントがありません"

#: src/modules/unpackfs/main.py:314
msgid "globalstorage does not contain a \"rootMountPoint\" key, doing nothing"
msgstr "globalstorage に \"rootMountPoint\" キーが含まれていません。何もしません。"

#: src/modules/unpackfs/main.py:319
msgid "Bad mount point for root partition"
msgstr "ルートパーティションのためのマウントポイントが不正です"

#: src/modules/unpackfs/main.py:320
msgid "rootMountPoint is \"{}\", which does not exist, doing nothing"
msgstr "ルートマウントポイントは \"{}\" ですが、存在しません。何もできません。"

#: src/modules/unpackfs/main.py:333 src/modules/unpackfs/main.py:340
#: src/modules/unpackfs/main.py:345
msgid "Bad unsquash configuration"
msgstr "unsquash の設定が不正です"

#: src/modules/unpackfs/main.py:334
msgid "The filesystem for \"{}\" ({}) is not supported"
msgstr "ファイルシステム \"{}\" ({}) はサポートされていません"

#: src/modules/unpackfs/main.py:341
msgid "The source filesystem \"{}\" does not exist"
msgstr "ソースファイルシステム  \"{}\" は存在しません"

#: src/modules/unpackfs/main.py:346
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "ターゲットシステムの宛先 \"{}\" はディレクトリではありません"

#: src/modules/displaymanager/main.py:380
msgid "Cannot write KDM configuration file"
msgstr "KDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:381
msgid "KDM config file {!s} does not exist"
msgstr "KDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:442
msgid "Cannot write LXDM configuration file"
msgstr "LXDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:443
msgid "LXDM config file {!s} does not exist"
msgstr "LXDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:517
msgid "Cannot write LightDM configuration file"
msgstr "LightDMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:518
msgid "LightDM config file {!s} does not exist"
msgstr "LightDM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:592
msgid "Cannot configure LightDM"
msgstr "LightDMの設定ができません"

#: src/modules/displaymanager/main.py:593
msgid "No LightDM greeter installed."
msgstr "LightDM greeter がインストールされていません。"

#: src/modules/displaymanager/main.py:624
msgid "Cannot write SLIM configuration file"
msgstr "SLIMの設定ファイルに書き込みができません"

#: src/modules/displaymanager/main.py:625
msgid "SLIM config file {!s} does not exist"
msgstr "SLIM 設定ファイル {!s} が存在しません"

#: src/modules/displaymanager/main.py:740
msgid "No display managers selected for the displaymanager module."
msgstr "ディスプレイマネージャが選択されていません。"

#: src/modules/displaymanager/main.py:741
msgid ""
"The displaymanagers list is empty or undefined in bothglobalstorage and "
"displaymanager.conf."
msgstr "ディスプレイマネージャのリストが bothglobalstorage 及び displaymanager.conf 内で空白か未定義です。"

#: src/modules/displaymanager/main.py:821
msgid "Display manager configuration was incomplete"
msgstr "ディスプレイマネージャの設定が不完全です"

#: src/modules/rawfs/main.py:35
msgid "Installing data."
msgstr "データのインストール。"

#: src/modules/services-openrc/main.py:38
msgid "Configure OpenRC services"
msgstr "OpenRCサービスを設定"

#: src/modules/services-openrc/main.py:66
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "ランレベル  {level!s} にサービス {name!s} が追加できません。"

#: src/modules/services-openrc/main.py:68
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "ランレベル  {level!s} からサービス {name!s} が削除できません。"

#: src/modules/services-openrc/main.py:70
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:103
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-openrc/main.py:110
msgid "Target runlevel does not exist"
msgstr "ターゲットとするランレベルは存在しません"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/services-openrc/main.py:119
msgid "Target service does not exist"
msgstr "ターゲットとするサービスは存在しません"

#: src/modules/services-openrc/main.py:120
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/plymouthcfg/main.py:36
msgid "Configure Plymouth theme"
msgstr "Plymouthテーマを設定"

#: src/modules/machineid/main.py:36
msgid "Generate machine-id."
msgstr "machine-id の生成"

#: src/modules/packages/main.py:62
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "パッケージの処理中 (%(count)d / %(total)d)"

#: src/modules/packages/main.py:64 src/modules/packages/main.py:74
msgid "Install packages."
msgstr "パッケージのインストール"

#: src/modules/packages/main.py:67
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] " %(num)d パッケージのインストール中。"

#: src/modules/packages/main.py:70
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] " %(num)d パッケージの削除中。"

#: src/modules/removeuser/main.py:34
msgid "Remove live user from target system"
msgstr ""

#: src/modules/dummypython/main.py:44
msgid "Dummy python job."
msgstr "Dummy python job."

#: src/modules/dummypython/main.py:97
msgid "Dummy python step {}"
msgstr "Dummy python step {}"
